cli/pack/common.go

```diff
@@ -257,6 +268,16 @@ func prepareBundle(cmdCtx *cmdcontext.CmdCtx, packCtx *PackCtx,
 	if err != nil {
 		return "", err
 	}
+
+	if !packCtx.CartridgeCompat {
```

Comment:
Why?

Reply:
It looks like that would imply that `cartridge-cli` would then be obligated to be able to perform integrity checks as well, and it is not going to.

I guess that throwing an error when both are specified would be a nicer behaviour.

Reply:
So let's return an error in this case. Looks like, now we silently skip signing.

Reply:
Now it throws an error. I've also added an integration test on tt-ee part